lib/lotus/cli.rb

```diff
@@ -1,110 +1,134 @@
+# TODO: Clarify/Questions
+# Create a migration along with the entity?
+# Reneame generate model => generate entity?
+# Naming slice/apps. I'd keep the slice generator, drop the app generator (slice can do everything app can)
+# Do we need to support namespaced models: lotus generate model foo/bar currently fails when loading the app
+# Can we remove the commands from the current namespace? The parent class is something that inherits from Thor.
```

Comment:
As a sidenote: Thor is used in the generators to process templates. But it is wrapped/hidden.